tools/gstf_preparation/gstf_preparation.py

```diff
@@ -429,10 +503,20 @@ def __main__():
                     print("Transcript '%s' in FASTA file '%s' not found in the gene feature information" % (transcript_id, fasta_arg), file=sys.stderr)
                     continue
 
-                if options.headers:
+                if options.headers == "transcript":
                     # Change the FASTA header to '>TranscriptId_species', as required by TreeBest
                     # Remove any underscore in the species
                     entry.header = ">%s_%s" % (transcript_id, species_for_transcript.replace('_', ''))
+                elif options.headers == "gene":
+                    # Change the FASTA header to '>GeneSymbol_species', as required by TreeBest
+                    # Remove any underscore in the species
+                    gene_name = fetch_gene_symbol_for_gene(conn, fetch_gene_id_for_transcript(conn, transcript_id))
+                    entry.header = ">%s-%s_%s" % (gene_name, transcript_id, species_for_transcript.replace('_', ''))
```

Comment:
Do you want GeneSymbol-TranscriptId_species (dash & underscore, as here) or GeneSymbol_TranscriptId_species (underscore & underscore, as in the select in the XML below)?

Reply:
Yes GeneSymbol_TranscriptId_species will break TreeBeST